code/src/components/LoadingIndicator.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+import styled from 'styled-components'
+
+export const LoadingIndicator = () => {
+  const isLoading = useSelector((store) => store.ui.isLoading)
+  return (
+    <>
```

Comment:
No need for React.Fragment here as well

Reply:
Oh oh, react asked me to use the fragment and shout out when I removed them.